frontend/public/components/_route.scss

```diff
@@ -15,3 +11,26 @@
 .co-m-route-tls-reveal__btn {
   padding-right: 0;
 }
+
```

Comment:
BEM? Just validating, there are a lot of `-`, that might be intentional.

Reply:
yeah, I need to ramp up on BEM, I'll check

Reply:
Good catch, it should probably be

```
.co-m-route__ingress-status
.co-m-route__ingress-status__conditions-heading
```

and drop the ingress-status-section class. Wondering how it looks like you just use h2/h3 instead, though? Do we need the custom style here?

Reply:
The custom style is to left-alight the table header and rows so that they appear under the 'Conditions' table header.

Reply:
Updated to BEM names @spadgett suggested.